app/question.js

```diff
@@ -13,7 +18,7 @@ define(['jquery', 'app/messages', 'app/config', 'app/string'], function ($, mess
   // If the user leaves tutorial mode, roll back to the first item.
   function checkTutorialStatus() {
     if (inTutorial) {
-      if (currentItemIndex == config.constant("NUMBER_TUTORIAL_QUESTIONS")) {
+      if (currentItemIndex == tutorialLength) {
```

Comment:
Use a function to describe what is happening, something like `hasFinishedTutorial()` maybe.